hono/test/utils/factory/items-outfits.ts

```diff
@@ -0,0 +1,40 @@
+import { faker } from '@faker-js/faker'
+import { drizzle } from 'drizzle-orm/postgres-js'
+import postgres from 'postgres'
+import * as schema from '../../../src/schema'
+import { itemTypeEnum, itemsToOutfits } from '../../../src/schema'
```

Comment:
Importing from '../../../src/schema' twice.

Reply:
This implementation already exists in a lot of places and is a best practice since `schema` defines the schema at large for use in the drizzle config, whereas the individual definitions (itemTypeEnum, etc.) is imported on a case by case basis.

Reply:
Good point, we can go with your approach